commit 900db66d3d03f75a412ab1184b5d7b6790c4f3af (refs/changes/86/1886/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-05-03T00:00:31+00:00 (2 years, 5 months ago)
    
    joyent/node-cueball#110 CHANGES.adoc was not updated for v2.2.6
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/CHANGES.adoc b/CHANGES.adoc
index 9eb3d07..5240d65 100644
--- a/CHANGES.adoc
+++ b/CHANGES.adoc
@@ -6,6 +6,17 @@ toc::[]
 
 ## v2.x
 
+### v2.2.6
+
+Maintenance release.
+
+Bugs fixed:
+
+ - #108 crash: unhandled smgr state transition -- issue which most commonly
+   happens due to HTTP requests that time out while stalled (buffers full)
+ - #109 Agent "health check ok" log msg needs domain name of pool (also adds
+   a bunch of other useful attributes to that log message)
+
 ### v2.2.5
 
 Maintenance release.
